### PR TITLE
Add troubleshooting tip

### PR DESCRIPTION
**Pull request by casconed**
Sorry to keep blowing up your docs with PRs - I was running this on a RN project with a ton of external deps and I was a little perplexed by my abysmal hit rate (~15%) until I tried bumping up the cache size and lo and behold that fixed it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +123,8 @@ iOS compile performance improvements of approximately 40-50% may be expected whe
 
 The _first_ build - the "cold" cache case - will be slower by around 15%, since buildcache has overhead to determine if it can use the cached object or not. On the cache miss case it then delegates to the compiler and stores the object for the next run which takes longer than a normal compile call.
 
+If you experience low cache hit rates on a project with a largely static codebase (i.e., one that should be very cacheable), try increasing your cache size.
```
Comment (mikehardy) [inline README.md]: What do you think about a little more of a "teach a person to fish" approach (check the log) vs just giving them the fish (increase cache)?

```suggestion
If you experience low cache hit rates on a project with a largely static codebase (i.e., one that should be very cacheable and 2 runs with nearly no changes do not cache for some reason), you should definitely increase the debugging level for the buildcache log and examine stats output before and after each run. A common and easy fix may be as simple as increasing your cache size.
```

**Review by mikehardy — COMMENTED** [review]
PRs are the lifeblood of an open repository, no reason to ever apologize for them I think? I really appreciate it anyway, especially in docs where you have fresh eyes while integrating it now and I have no perspective any more :-)

Tell me what you think about my comment and the docs will be even better post-merge

File: README.md (hunk 1 above)
Comment (casconed) [reply README.md]: Ah yeah, great edit!

**Comment by mikehardy** [comment]
Slight aside, I just noticed your profile lists you in Oakland, CA. I still have a 415 number from the dot-com boom but 510 (upper laurel is probably what the real estate agents call it now) was my home for around a decade :-). Cheers

**Review by mikehardy — APPROVED** [review]

**Comment by casconed** [comment]
I actually fled Oakland in 2015 - I was down by Lake Merritt from 2007 until 2012, then West Oakland until I left. Definitely miss a lot of things about living there!

**Pull request merged**